src/index.ts

```diff
@@ -178,12 +180,15 @@ export class Linkurious {
     traits?: any;
     timestamp?: string;
     context?: any;
-  }): Promise<void> {
-    return this._fetcher.fetch({
-      url: '/analytics',
-      method: 'POST',
-      body: data,
-    });
+  }): Promise<Success<void> | Rejection> {
+    return this._fetcher
+      .fetch({
+        url: '/analytics',
+        method: 'POST',
+        body: data,
+      })
+      .then(() => new Success(undefined))
```

Comment:
Change `Success` signature to make `response` argument optional. If a API return a `Success<void>`, you will not have to do `new Success(undefined)`